docs/spelling_wordlist

```diff
@@ -23,8 +23,11 @@ Ctrl
 dialogs
 Django
 draggable
+Flatpak
 Flexbox
 GBulb
+GeoClue
+Geoclue
```

Comment:
+1 to using the "official" capitalisation; do we need to preserve the `Geoclue` form?